src/schema/common.cljs

```diff
@@ -11,6 +11,14 @@
 (def ^:private ?public-key
   [:re #"^0x04[0-9a-f]{128}$"])
 
+(def ^:private ?image-source
+  [:or
+   [:string]
+   [:int]
+   [:map
+    [:uri [:maybe [:string]]]
+    [:require [:maybe [:string]]]]])
```

Comment:
AFAIK, an image source for react native can't be a string, it should always be an int or object (map, for us).

So other devs might see this schema and wrongly use it for their react native image sources.

Reply:
@ulisesmac You are right. I have updated the implementation. Please have a look.

Thanks!